### PR TITLE
Add dynamic mob-nav-menu

### PR DESCRIPTION
**Pull request by Hope1226**
In this feature branch we:
- Created script.js file
- Set up the lintters for JS.
- Created dynamic navigation menu (DVM) for mob version
- Added style to the DVM

### CODE REVIEW

File: Assets/stylesheet/style.css [hunk 1]
```diff
@@ -25,6 +25,70 @@ html {
...
+  align-items: start;
+}
+
+.mob-navigation ul li {
+  padding: 20px 0;
+  border-bottom: 1px solid #6f6c6b;
+}
+
+.menu-open {
+  display: flex;
```
Comment (GzimAsani) [inline Assets/stylesheet/style.css:89]: - [ ] You have used `Display: flex` 12 times, this is falls under the don't repeat rule. I would suggest you create a new class just for display: flex and then give to any class that you need in HTML

File: index.html [hunk 2]
```diff
@@ -26,19 +26,34 @@
         gtag('js', new Date());
        gtag('config', 'G-8JBDZTMFBC');
   </script>
+  <script src="Script/script.js" defer></script>
   <title>Hope's portfolio</title>
 </head>
 <body>
```
Comment (GzimAsani) [inline index.html:32]: - [ ] Please check the original version and try to change yours in order to look more similar(Give some margin down)
**Original**:
Screenshot from 2021-10-04 21-24-39
**Yours**: 
Screenshot from 2021-10-04 21-26-28

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @Hope1226 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good use of gitflow :+1: 

## Required Changes ♻️
- [ ] _Check the comments under the review for more information._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZeenatLawal — APPROVED** [review]
Hi @Hope1226 and @sj1978, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**